Update the influent crate from 0.4.0 to 0.4.1

SimonSapin opened the pull request:
The tarball for 0.4.1 is 44 KB uncompressed and contains 18 files.

The tarball for 0.4.0 is 45 MB uncompressed and contains 22113 files, most of which appear to be the output of `cargo doc` that was accidentally included in a release.

CC https://github.com/servo/servo/pull/21863#issuecomment-426609991

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/servo/servo/21865)
<!-- Reviewable:end -->

SimonSapin committed:
Update the influent crate from 0.4.0 to 0.4.1

The tarball for 0.4.1 is 44 KB uncompressed and contains 18 files.

The tarball for 0.4.0 is 45 MB uncompressed and contains 22113 files,
most of which appear to be the output of `cargo doc` that was accidentally
included in a release.

SimonSapin commented:
@bors-servo r+

nox commented:
@bors-servo r+

bors-servo commented:
:bulb: This pull request was already approved, no need to approve it again.

- There's another pull request that is currently being tested, blocking this pull request: #21850

bors-servo commented:
:pushpin: Commit 8b43719 has been approved by `nox`

<!-- @bors-servo r=nox 8b43719baa28d5e7984844258d9683d48f5091bd -->

bors-servo commented:
:hourglass: Testing commit 8b43719baa28d5e7984844258d9683d48f5091bd with merge 726cf0b48c2fd1b26b970982cc70f3a89e692b30...

bors-servo commented:
:broken_heart: Test failed - [android](http://build.servo.org/builders/android/builds/12704)

SimonSapin commented:
```
Downloading GStreamer dependencies
[…]
BadZipfile: File is not a zip file
```

Network glitch?

@bors-servo retry

bors-servo commented:
:bomb: Failed to start rebuilding: `Unknown error`

bors-servo commented:
:hourglass: Testing commit 8b43719baa28d5e7984844258d9683d48f5091bd with merge 5b64d4688d4a896d25edb45d1f006be37cfbf2db...

bors-servo commented:
:broken_heart: Test failed - [android](http://build.servo.org/builders/android/builds/12706)

SimonSapin commented:
I can reproduce locally. https://github.com/servo/libgstreamer_android_gen/blob/master/out/gst-build-armeabi-v7a.zip?raw=true is 404. CC @ferjm

CYBAI commented:
Maybe the failure is related to #21864 which @ferjm is working on?

ferjm commented:
@bors-servo retry

bors-servo commented:
:bomb: Failed to start rebuilding: `Unknown error`

bors-servo commented:
:hourglass: Testing commit 8b43719baa28d5e7984844258d9683d48f5091bd with merge 8ee40923ac8c997a41f89d25f91805d9d8c9b680...

SimonSapin committed:
Work around removal of remote files used by the build system

https://github.com/servo/libgstreamer_android_gen/commit/f5a832741569c96b57dc2b964df9ae37016454c2

SimonSapin commented:
I’ve pointed mach to a previous version of libgstreamer_android_gen.

@bors-servo r+

bors-servo commented:
:pushpin: Commit d2ee766 has been approved by `SimonSapin`

<!-- @bors-servo r=SimonSapin d2ee7663f3172646df0f56d3ede138c7d3c8b268 -->

bors-servo commented:
:hourglass: Testing commit d2ee7663f3172646df0f56d3ede138c7d3c8b268 with merge e56db1f322b504d9661a60127b74fb8e0bfc9669...

bors-servo commented:
:sunny: Test successful - [android](http://build.servo.org/builders/android/builds/12708), [android-x86](http://build.servo.org/builders/android-x86/builds/823), [arm32](http://build.servo.org/builders/arm32/builds/12892), [arm64](http://build.servo.org/builders/arm64/builds/12931), [linux-dev](http://build.servo.org/builders/linux-dev/builds/13003), [linux-rel-css](http://build.servo.org/builders/linux-rel-css/builds/10013), [linux-rel-wpt](http://build.servo.org/builders/linux-rel-wpt/builds/10033), [mac-dev-unit](http://build.servo.org/builders/mac-dev-unit/builds/13026), [mac-rel-css1](http://build.servo.org/builders/mac-rel-css1/builds/5480), [mac-rel-css2](http://build.servo.org/builders/mac-rel-css2/builds/5485), [mac-rel-wpt1](http://build.servo.org/builders/mac-rel-wpt1/builds/9743), [mac-rel-wpt2](http://build.servo.org/builders/mac-rel-wpt2/builds/9795), [mac-rel-wpt3](http://build.servo.org/builders/mac-rel-wpt3/builds/5549), [mac-rel-wpt4](http://build.servo.org/builders/mac-rel-wpt4/builds/5487), [status-taskcluster](https://tools.taskcluster.net/task-group-inspector/#/IWQ8e7azSOeIorJN9LUG5g), [windows-msvc-dev](http://build.servo.org/builders/windows-msvc-dev/builds/8387)
Approved by: SimonSapin
Pushing e56db1f322b504d9661a60127b74fb8e0bfc9669 to master...
